### PR TITLE
Ruby on Rails Microverse Forms

### PR DESCRIPTION
**Pull request by rkyankya**
Created the project as described for user creation and editing.

Following the instructions of the Odin project, we only created the routes for editing and creating (new, edit, create, and update) and we didn't change the path, thus the only working paths are "/users/new" and "/users/:id/edit". 

The root rout gives you the list of the user's username and email, so you can have a visual aid for the user's information without having to rely on the rails console.

### CODE REVIEW

**Review by Ispirett — CHANGES_REQUESTED** [review]
## Status Changes Required! :+1: 
Great  Job so far @rkyankya  @lucasmazo32 
There is just one thing missing.
- [ ] You need to have a plain html form also [check](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form1.png)

File: app/views/users/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<form accept-charset="UTF-8" action="/users" data-remote="true" method="post">
+  <input type="hidden" name="authenticity_token" value="NRkFyRWxdYNfUg7vYxLOp2SLf93lvnl+QwDWorR42Dp6yZXPhHEb6arhDOIWcqGit8jfnrPwL781/xlrzj63TA==" />
```
Comment (ebukaume) [inline app/views/users/edit.html.erb]: Authenticity token should be dynamically generated

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<div class="jumbotron jumbotron-fluid">
+  <div class="container">
+    <form accept-charset="UTF-8" action="/users" data-remote="true" method="post">
+      <input type="hidden" name="authenticity_token" value="NRkFyRWxdYNfUg7vYxLOp2SLf93lvnl+QwDWorR42Dp6yZXPhHEb6arhDOIWcqGit8jfnrPwL781/xlrzj63TA==" />
```
Comment (ebukaume) [inline app/views/users/new.html.erb]: Same as above

**Review by ebukaume — CHANGES_REQUESTED** [review]
## Changes Required! :stop_sign: 

Hi @rkyankya and @lucasmazo32 :wave: 

Nice work so far, but there are a few things we need to take care of. 

* [x] It is possible to create a new user in app. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/create_new_user.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))
* [x] it is possible to edit a user in app. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/edit_user.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))

_I was unable to do either. Did you test the app?_

**For your views, you need to have:**
- [x] Commented out `form`, ([screenshot 1](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form1.png) [screenshot 2](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form2.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))

- [x] Commented out `form_tag` that uses `label_tag`, `text_field_tag`, `email_field_tag`, `password_field_tag` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_tag2.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))

- [x] `form_for` that uses `@user` from controller ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_for.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))

- [x] The same applies, also, to `edit` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_edit.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))

**Kindly submit another code review after fixing these issues**

Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

**Review by Ispirett — APPROVED** [review]
Status Approved 👍 
Great @rkyankya  @ebukaume  work you can merge now!

**Pull request merged**